antsibull/data/sphinx_init/requirements_txt.j2

```diff
@@ -0,0 +1,4 @@
+antsibull
+ansible-pygments
+sphinx
+sphinx_rtd_theme
```

Comment:
Why use this theme and not Ansible? Also, if you're looking for an alternative one, I'd recommend Furo — it's probably the only one that is adaptive, mobile-friendly, well-maintained and accessible. It's by a pip maintainer and it gained a lot of popularity during the past year.